.github/workflows/reusable_testing.yml

```diff
@@ -0,0 +1,52 @@
+name: reusable_testing
+
+on:
+  workflow_call:
+    inputs:
+      artifact_name:
+        description: The name of the Tarantool build artifact
+        default: ubuntu-focal
+        required: false
```

Comment:
Is `ubuntu-focal` a valid artifact name? It seems that it isn't.

Reply:
Indeed, it's the same as in other similar workflows. Okay. https://github.com/tarantool/expirationd/blob/master/.github/workflows/reusable_testing.yml